operator/pkg/reconciliation/construct_statefulset.go

```diff
@@ -80,6 +79,29 @@ func shouldDefineSecurityContext(dc *api.CassandraDatacenter) bool {
 	return dc.Spec.ServerType == "dse" || images.CalculateDockerImageRunsAsCassandra(dc.Spec.ServerVersion)
 }
 
+func rackNodeAffinitylabels(dc *api.CassandraDatacenter, rackName string) (nodeAffinityLabels map[string]string,
+	Error error) {
```

Comment:
we have favored not using named return values

Reply:
it avoids to define them but as you wish